src/styles/themes.ts

```diff
@@ -53,9 +53,9 @@ export const lightTheme: DefaultTheme = {
   global: {
     accent: '#5981f3',
     complementary: '#ce5cf8',
-    alert: '#ed4a34',
+    alert: '#ee5959',
     warning: '#ffa600',
-    valid: '#0ea68f',
+    valid: '#00a661',
```

Comment:
Should we port these colors in our other apps as well?

Reply:
Not yet :) I'm making small adjustments here and there, as I feel like a theme refactor will be done in the future (more granular variables etc.).